app/serializer/city_serializer.rb

```diff
@@ -0,0 +1,13 @@
+class CitySerializer < Blueprinter::Base
```

Comment:
Are we testing serializers? Is there a way to do it?

Reply:
I have added thanks to this comment the tests for the `Country` and `City` serializer